libs/net/dal/Services/ContentService.cs

```diff
@@ -11,11 +12,19 @@ namespace TNO.DAL.Services;
 public class ContentService : BaseService<Content, long>, IContentService
```

Comment:
I don't understand why this is in the ContentService?  It's the SyndicationService that imported the invalid character.  We should be resolving the issue at its source.  Update the SyndicationService to fix invalid characters when it ingests content.

Reply:
This should be updated now, thanks